docs/source/notebooks/tpt.ipynb

```diff
@@ -0,0 +1,779 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "paths, capacities = flux.pathways(fraction=.3, maxiter=1000)\n",
+    "print(len(paths))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we can represent $30\\%$ of the total flux with four pathways."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib as mpl\n",
+    "import matplotlib.pyplot as plt\n",
+    "from mpl_toolkits.axes_grid1 import make_axes_locatable\n",
+    "\n",
+    "fig, ax = plt.subplots(figsize=(10, 10))\n",
+    "\n",
+    "ax.scatter(*start, marker='*', label='Start', c='cyan', s=150, zorder=5)\n",
+    "\n",
+    "for capacity, path in zip((capacities / np.array(capacities).sum())[:10], paths[:10]):\n",
+    "    path = np.array([sim.state_to_coordinate(state) for state in path])\n",
+    "    sim.plot_path(ax, path, lw=capacity*10, intermediates=False, \n",
+    "                  color_lerp=False, label=f\"Capacity {capacity:.2e}\")\n",
+    "    ax.scatter(*path.T, marker='x')\n",
+    "\n",
+    "handles, labels = sim.plot_2d_map(ax)\n",
+    "ax.legend(handles=handles, labels=labels);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example from trajectories\n",
+    "\n",
+    "Using the same example we can generate timeseries which can be used to re-estimate a Markov state model and perform a TPT analysis. To this end, we simulate 1000 trajectories all starting from the same point and return the micro states directly rather than $(i, j)$ coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trajs = []\n",
+    "for _ in range(1000):\n",
+    "    trajs.append(sim.walk(start=start, n_steps=2000, return_states=True, stop=False))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From these trajectories we can count state transitions based on the prior knowledge that there are actually $100$ states and select the largest connected submodel - which due to the barriers does not contain all the states."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "count_model = sktime.markov.TransitionCountEstimator(1, 'sliding', n_states=sim.n_states) \\\n",
+    "    .fit(trajs).fetch_model()\n",
+    "count_model = count_model.submodel_largest()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.plot(count_model.state_symbols, 'o')\n",
```

Comment:
i don't really get what this is supposed to show me

Reply:
just that there are some gaps in the state symbols, replaced it with a print statement now